components/MoveToTop.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+import {FaArrowCircleUp} from 'react-icons/fa';
+
+const ScrollButton = () =>{
```

Comment:
Keep this file in basic folder..

Reply:
means..?

Reply:
i mean to say, in components folder we have a basic folder.. move this file inside that folder

Reply:
done